libraries/imgui-opencv-poco/.gitignore

```diff
@@ -0,0 +1,372 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There is a .gitignore on root level of this project. It could be move to there.

Reply:
The root .gitignore is not filtering a lot of stuff that this project is generating, it is good to leave this here (hierarchical .gitignores are fine).